postreise/plot/plot_capacity_map.py

```diff
@@ -50,34 +50,26 @@ def map_plant_capacity(
     if not isinstance(scenario, Scenario):
         raise TypeError(f"scenario must be a {Scenario} object")
 
-    p = figure(
-        tools="pan,wheel_zoom,reset,save",
-        x_axis_location=None,
-        y_axis_location=None,
-        plot_width=figsize[0],
-        plot_height=figsize[1],
-        output_backend="webgl",
-        sizing_mode="scale_both",
-        match_aspect=True,
-        x_range=x_range,
-        y_range=y_range,
-    )
+    canvas = create_map_canvas(figsize=figsize, x_range=x_range, y_range=y_range)
 
     # state borders
-    default_plot_states_kwargs = {
+    default_state_borders_kwargs = {
         "line_color": "gray",
         "line_width": 2,
         "fill_alpha": 0,
         "background_map": True,
     }
-    if plot_states_kwargs is not None:
-        all_plot_states_kwargs = {**default_plot_states_kwargs, **plot_states_kwargs}
+    if state_borders_kwargs is not None:
```

Comment:
Using a ternary operator here can save you a couple of lines:
```
all_state_borders_kwargs = (
    {**default_state_borders_kwargs, **state_borders_kwargs}
    if state_borders_kwargs is not None
    else default_state_border_kwargs
)
```

Reply:
Done